README.md

```diff
@@ -35,21 +36,42 @@ docker run --rm -it --cap-add "SYS_ADMIN" harrisonai/cobalt-rootless-nvidia-dind
 
 As a Kubernetes deployment:
 
-```apiVersion: apps/v1
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nvidia-devices
+
+---
+
+apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: dind
+  name: test
 spec:
   selector:
     matchLabels:
-      app: dind
+      app: test
   template:
     metadata:
       annotations:
         container.apparmor.security.beta.kubernetes.io/dind: unconfined
       labels:
-        app: dind
+        app: test
     spec:
+      containers:
+        - name: main
+          image: ubuntu:latest
+          command:
+            - sleep
```

Comment:
As an example perhaps running `nvidia-smi` would be useful? Given an appropriate base image

Reply:
(really no boggy, this is chill)